docs/source/user-guide/tabular/tutorials/plot_quick_train_test_validation.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+"""
+Train Test Validation Scenario on Lending Club Data - Quickstart
+*********************************************************************
+
+The deepchecks train test validation suite is relevant any time you wish to 
+validate two data subsets. For example:
+
+- Comparing distributions across different train-test splits (e.g. before 
+  training a model or when splitting data for cross-validation)
+- Comparing a new data batch to previous data batches
+
+Here we'll use a loan's dataset 
+(:mod:`deepchecks.tabular.datasets.classification.lending_club`),
+to demonstrate how you can run the suite with only a few simple lines of code, 
+and see which kind of insights it can find.
+
+.. code-block:: bash
+
+    # Before we start, if you don't have deepchecks installed yet,
+    # make sure to run:
+    pip install deepchecks -U --quiet #--user
+"""
+
+#%%
+# Load Data and Prepare Data
+# ====================================================
+#
+# Load Data
+# -----------
+
+
+from deepchecks.tabular.datasets.classification import lending_club
+import pandas as pd
+
+data = lending_club.load_data(data_format='Dataframe', as_train_test=False)
+data.head(2)
+
+
+
+#%%
+# Split Data
+# -------------
+
+# convert date column to datetime, `issue_d`` is date column
+data['issue_d'] = pd.to_datetime(data['issue_d'])
+
+# Use data from June and July for train and August for test:
+train_df = data[data['issue_d'].dt.month.isin([6, 7])]
+test_df = data[data['issue_d'].dt.month.isin([8])]
+
+
+#%%
+# Run Deepchecks for Train Test Validation
+# ===========================================
+#
+# Define a Dataset Object
+# -------------------------
+#
+# Create a deepchecks Dataset, including the relevant metadata (label, date, index, etc.).
+# Check out :class:`deepchecks.tabular.Dataset` to see all of the columns that can be declared.
```

Comment:
```suggestion
# Check out :class:`deepchecks.tabular.Dataset` to see all of the inputs that can be declared.
```
Not sure about "inputs", but not only columns (e.g. label_type)

Reply:
changed to columns and types